docs/auth.md

```diff
@@ -103,3 +113,19 @@ Like the other OAuth2 providers, access to Chronograf via Heroku can be restrict
 ```sh
 export HEROKU_ORGS=hill-valley-preservation-sociey,the-pinheads
 ```
+
+### Optional: Configuring Authentication Duration
+
+By default, auth will remain valid for 30 days via a cookie stored in the browser. This duration can be changed with the environmental variable `AUTH_DURATION`. For example, to change it to 1 hour, use:
+
+```sh
+export AUTH_DURATION=1h
+```
+
+The duration uses the golang [time duration format](https://golang.org/pkg/time/#ParseDuration), so the largest time unit is `h` (hours). So to change it to 45 days, use:
+
+```sh
+export AUTH_DURATION=1080h
+```
+
+Additionally, for greater security, if you want to require re-authentication every time the browser is closed, set `AUTH_DURATION` to `0`. This will make the cookie transient (aka "in-memory").
```

Comment:
Hm.. definitely cleaner language, but same thoughts from here as about the above suggestion of removing explicit security implementation details.